pkg/resourcemanager/controller/garbagecollector/reconciler_test.go

```diff
@@ -75,7 +74,6 @@ var _ = Describe("Collector", func() {
 			labeledSecret2       *corev1.Secret
 			labeledSecret3       *corev1.Secret
 			labeledSecret4       *corev1.Secret
-			labeledSecret5       *corev1.Secret
```

Comment:
Not necessary but it would be nice if you keep `labeledSecret5` and remove `labeledSecret10`.